### PR TITLE
Add install guide for Xenial build env

### PR DESCRIPTION
**Pull request by schultyy**
This PR adds an install guide for the new Xenial build environment

### CODE REVIEW

File: user/enterprise/xenial.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+---
+
+## System Setup
+
+**Platform Requirements**: To use the Xenial build containers, the Travis CI installation must be at 2.2.6 or higher. Please be sure to [upgrade](/user/enterprise/upgrading/), if needed, before getting started.
+
+**Worker Requirements**:
+
+We recommend using AWS's `c4.2xlarge` as the instance type with at least 40gb of disk space, running Ubuntu 16.04 or later. Port 22 must be open for SSH during installation and operation.
```
Comment (danishkhan) [inline user/enterprise/xenial.md]: We should avoid making it seem as if only AWS is supported for Travis CI Enterprise.

We recommend using a machine with 8 vCPUs and 15 GiB of memory and at least 40 GiB of disk space. If you're using AWS that will be their `c4.2xlarge` instance type. Also, you'll want to run Ubuntu 16.04 or later.

Comment (schultyy) [reply user/enterprise/xenial.md]: Makes sense to me. I edited that in this file and same also for Trusty ✅

File: user/enterprise/trusty.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ layout: en_enterprise
 
 **Worker Requirements**:
 
-We recommend using AWS's `c4.2xlarge` as the instance type with at least 40gb of disk space. Port 22 must be open for SSH during installation and operation.
+We recommend using a machine with 8 vCPUs and 15 GiB of memory and at least 40 GiB of disk space. If you're using AWS that will be their c4.2xlarge instance type. Also, you'll want to run Ubuntu 16.04 or later. Port 22 must be open for SSH during installation and operation.
 
 In addition, _Precise build containers and Trusty build containers must be on different instances_. To run both Precise and Trusty builds, at least two worker instances are required.
```
Comment (plaindocs) [inline user/enterprise/trusty.md]:
```suggestion
> _Precise build containers and Trusty build containers must be on different instances_. To run both Precise and Trusty builds, at least two worker instances are required.
```

File: user/enterprise/xenial.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## System Setup
+
+**Platform Requirements**: To use the Xenial build containers, the Travis CI installation must be at 2.2.6 or higher. Please be sure to [upgrade](/user/enterprise/upgrading/), if needed, before getting started.
+
+**Worker Requirements**:
+
+We recommend using a machine with 8 vCPUs and 15 GiB of memory and at least 40 GiB of disk space. If you're using AWS that will be their c4.2xlarge instance type. Also, you'll want to run Ubuntu 16.04 or later. Port 22 must be open for SSH during installation and operation.
+
+In addition, _Trusty and Xenial build containers must be on different instances_. To run both Trusty and Xenial builds, at least two worker instances are required.
```
Comment (plaindocs) [inline user/enterprise/xenial.md]:
```suggestion
> _Trusty and Xenial build containers must be on different instances_. To run both Trusty and Xenial builds, at least two worker instances are required.
```

**Pull request merged**